test/conformance/kuttl/mutate/clusterpolicy/standard/existing/background-false/README.md

```diff
@@ -0,0 +1,3 @@
+# Title
+
+This is a test for mutation of existing resources when background is set to false. The mutateExisting policy does not require `.spec.background=true` to be applied.
```

Comment:
Please use the README template provided in the scaffold dir.

Reply:
Done!